packages/drivers/driver-base/package.json

```diff
@@ -57,6 +57,10 @@
   },
   "typeValidation": {
     "version": "2.0.0",
-    "broken": {}
+    "broken": {
+      "ClassDeclaration_DocumentDeltaConnection": {
+        "backCompat": false
```

Comment:
FYI: this is due to public docmentId -> protected relaySessionId conversion. I do not expect anybody to derive from these classes (trying to avoid pushing to next branch).

Reply:
you are removing a public property. this seems like an obvious breaking change. even if no one uses it, the type will be incompatible so version mixing will be a problem. i don't think you can do this in main.